suite-common/wallet-core/src/fiat-rates/fiatRatesReducer.ts

```diff
@@ -96,6 +96,19 @@ export const prepareFiatRatesReducer = createReducerWithExtraDeps(
                     };
                 }
             })
+            .addCase(updateFiatRatesThunk.rejected, (state, action) => {
```

Comment:
error is normally handled per ticker in fullfilled case, but in cases somethign goes really wrong like here we want to at least have error in redux